controllers/infinispan_config_controller.go

```diff
@@ -82,6 +82,12 @@ func (reconciler *ConfigReconciler) Reconcile(ctx context.Context, request recon
 		return reconcile.Result{}, fmt.Errorf("unable to fetch Infinispan CR %w", err)
 	}
 
+	// Don't reconcile Infinispan CRs marked for deletion
+	if infinispan.GetDeletionTimestamp() != nil {
```

Comment:
Should we check if deletion timestamp is in the past? Seems it could be a future value

Reply:
Right... I should have read after the first sentence of the docs :sweat_smile: